backend/src/packages/users/users.ts

```diff
@@ -6,7 +7,8 @@ import { UserRepository } from './user.repository.js';
 import { UserService } from './user.service.js';
 
 const userRepository = new UserRepository(UserModel);
-const userService = new UserService(userRepository);
+const config = new BaseConfig(logger);
```

Comment:
Import config from #libs/packages/config/config.ts, do not create a new instance